shared/coordinator/src/coordinator.rs

```diff
@@ -85,6 +85,7 @@ pub enum ClientState {
     TS,
 )]
 #[serde(bound = "I: NodeIdentity")]
+#[repr(C)]
```

Comment:
This is an actual change (as the Repr(Rust) was the default here)

However and this is important: both `Repr(Rust)` and `Rust(C)` have the exact same binary outcome here, by pure luck. So it is NOT a breaking change, it just correctly instructs the IDL to use the Repr(C) instead of the default Repr(Rust) of which the behaviour is undefined and compiled dependent.